zebra-rpc/src/methods.rs

```diff
@@ -3918,10 +3986,11 @@ impl Utxo {
     }
 }
 
-/// A struct to use as parameter of the `getaddresstxids`.
+/// A struct to use as parameter of `getaddresstxids`
 ///
 /// See the notes for the [`Rpc::get_address_tx_ids` method].
 #[derive(Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize, Getters, new)]
+#[serde(from = "DGetAddressTxIdsRequest")]
 pub struct GetAddressTxIdsRequest {
     // A list of addresses to get transactions from.
```

Comment:
Optional/Nitpick: Just noticing that this is a regular comment instead of a doc comment. Could add some more detail here (other fields are also regular comments):

```suggestion
    /// A list of addresses. The RPC method will get transactions IDs that sent or received 
    /// funds to or from these addresses.
```